roles/capture-agent-common/tasks/set_facts.yml

```diff
@@ -131,17 +131,6 @@
     GC_line_in_source: "{{ line_in_source.stdout }}"
   when: line_in_source.stdout != '' and inventory_hostname in groups['capture-agents-v4l2-split']
 
-- name: "If Staging, Set Peakaboo Server"
-  set_fact:
-    peakaboo_meteor: "{{ peakaboo_staging_meteor }}"
-    peakaboo_http: "{{ peakaboo_staging_http }}"
-  when: deploy_environment is defined and deploy_environment == 'staging'
```

Comment:
is this `when` required?

Reply:
Umm.. that chunk is all being removed as unnecessary (we don't use the staging option)

Reply:
doh!